frontend/src/metabase/ui/components/inputs/Select/Select.module.css

```diff
@@ -9,7 +9,6 @@
 }
 
 .SelectWrapper {
-  margin-top: var(--mantine-spacing-xs);
```

Comment:
This is actually handled in some rules below that add margin if a lable, error, or description is included